source/stylesheets/_banner.scss

```diff
@@ -0,0 +1,37 @@
+$_banner-background-color: #e7f1ec;
```

Comment:
Thoughts on `#e7f1ec` and `#f55481` (below) going within the global variables partial?